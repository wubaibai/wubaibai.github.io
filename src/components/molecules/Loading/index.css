.loading {
	display: flex;
	width: 100vw;
	height: 100vh;
	background: var(--blackBlue);
	justify-content: center;
	align-items: center;
}
