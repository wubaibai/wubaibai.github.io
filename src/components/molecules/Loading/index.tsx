import React from 'react';

import Logo from 'components/atoms/Logo';

import style from './index.css';

const Loading: React.FC = () => (<div className={style.loading}>
    <Logo animate={true} />
</div>);

export default Loading;
