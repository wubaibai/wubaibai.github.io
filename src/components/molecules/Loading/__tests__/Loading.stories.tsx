import React from 'react';
import { storiesOf } from '@storybook/react';

import Loading from 'components/molecules/Loading';

const stories = storiesOf('molecules/Loading', module);

stories.add('__interactive', () => <Loading />, { jest: 'Loading' });
