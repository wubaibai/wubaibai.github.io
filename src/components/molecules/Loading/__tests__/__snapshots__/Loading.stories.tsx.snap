// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__interactive 1`] = `
<DocumentFragment>
  <div
    class="loading"
  >
    <div
      class="logo animate"
      id="logo"
    >
      <div
        class="up"
      />
      <div
        class="down"
      />
    </div>
  </div>
</DocumentFragment>
`;
